### PR TITLE
Add a Teleport Team scope

### PR DESCRIPTION
**Pull request by ptgott**
This change adds Teleport Team support to:
- The scope icons on the left navigation sidebar that indicate the relevance of a particular page
- The scope switcher on the upper navigation bar

This change is not intended to fine-tune the look and feel of these components, but to do the minimum required work for us to be able to document Team-specific functionality. As a result:
- The Team icon is a dark gray, the same color as the OSS icon
- The scope switcher component is now 25% wider than it used to be, causing issues on mobile (though the component is still visible on mobile screen sizes)

The intention here is that we can correct these issues in later changes.

Note that, prior to this change, the `forScopes` field in the docs config (`docs/config.json` in content submodules) supported string values with a format similar to `"enterprise,cloud,oss"`.

However, it was difficult to extend this functionality to accommodate Teleport Team without adding new parsing logic, which was beyond the scope of this change. Since we have never used this functionality in a docs configuration file, removing it eliminates unnecessary complexity in parsing the docs config.

### CODE REVIEW

**Comment by ptgott** [comment]
Adding a "Do not merge" label until we have content PRs ready to merge to include Team-related content in the docs. Otherwise, we'll have a scope switcher option for nonexistent content.

**Comment by alexfornuto** [comment]
While there are [separate issues](https://github.com/gravitational/docs/issues/302) with the behavior, the cloud option disables the version drop-down. Should the team option do the same?

Tested ScopedBlocks and Tabs, all work correctly with the new scope 👍

**Comment by ptgott** [comment]
@alexfornuto Thanks for testing! I've pushed a change that disables version switching for the `team` scope

**Review by avatus — APPROVED** [review]

**Pull request merged**
